core/modules/commands/savewikifolder.js

```diff
@@ -93,10 +114,13 @@ WikiFolderMaker.prototype.save = function() {
 						self.log("Adding built-in plugin: " + libraryDetails.name);
 						newWikiInfo[libraryDetails.type] = newWikiInfo[libraryDetails.type]  || [];
 						$tw.utils.pushTop(newWikiInfo[libraryDetails.type],libraryDetails.name);
-					} else {
+					} else if(self.explodePlugins == "yes") {
 						// A custom plugin
 						self.log("Processing custom plugin: " + title);
 						self.saveCustomPlugin(tiddler);
+					} else if(self.explodePlugins == "no") {
```

Comment:
It is problematic to test for both "yes" and "no" because of ambiguity when the value is neither "yes" nor "no".  In these situations, I would normally only test for the non-default value, so we'd be testing `if(self.explodePlugins !== "no")` etc.

Also please note that the core coding style is to use triple equals signs for comparisons.

Reply:
okk